roles/ceph-common/tasks/pre_requisites/prerequisite_rh_storage_iso_install.yml

```diff
@@ -6,7 +6,13 @@
   with_items:
     - "{{ ceph_stable_rh_storage_mount_path }}"
     - "{{ ceph_stable_rh_storage_repository_path }}"
-    - "{{ ceph_stable_rh_storage_iso_path }}"
+
+- name: ensure destination iso directory exists
+  file:
+    path: "{{ ceph_stable_rh_storage_iso_path | dirname }}"
+    state: directory
+    recurse: yes
+  when: "'{{ ceph_stable_rh_storage_iso_path | dirname }}' != '/'"
```

Comment:
My inclination was not to even attempt it if not necessary. This is totally not an issue if the path is `/tmp/`  (but it was when the path was `/root/iso/`)

I don't mind removing the when clause if you feel it is redundant